src/components/pages/project-landscape/faq/faq.jsx

```diff
@@ -12,11 +12,12 @@ const Requirements = () => (
         license. Any documentation must be licensed under an open license.
       </li>
       <li>
-        The project must be using eBPF as its underlying core technology, in other words, a project
-        would lose its purpose if the eBPF parts are removed.
+        The project must be using eBPF as its underlying core technology and help accelerate the
+        adoption of eBPF in production. In other words, a project would lose its purpose if the 
+        eBPF parts are removed.
```

Comment:
```suggestion
        The project must be using eBPF as its underlying core technology (in other words, a project would lose its purpose if the 
        eBPF parts are removed) or help accelerate the adoption of eBPF in production.
```
The point is to broaden, not narrow, the scope so that things like LLVM and gcc are ok to list.

Reply:
ahh that makes sense, thanks for the clarification

Reply:
Agree, with the suggestion this looks great to me, thanks Dave!